build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -123,6 +127,18 @@ abstract class BuildCommitDistribution @Inject internal constructor(
         }
     }
 
+    fun copyToFinalDestination(checkoutDir: File) {
+        fsOps.copy {
```

Comment:
Why do we use `fs.copy` here if we only copy one file? `File.copy` would be simpler in this case, wouldn't it?

Reply:
Yes, I use Kotlin `copyTo` instead.